src/frontend/src/icp/components/send/IcSendModal.svelte

```diff
@@ -177,6 +179,11 @@
 	setContext<EthereumFeeContextType>(ETHEREUM_FEE_CONTEXT_KEY, {
 		store: initEthereumFeeStore()
 	});
+
+	const goToWizardStep = (stepName: WizardStepsSend) => {
```

Comment:
Should we extract this in a utils to avoid duplicated code?

Reply:
Yes, agreed! I was going to do it in the next PR eheh

Reply:
Changing it with PR #1378